How to Find and Use Technical Resources if You’re New to Tech:proposal.md

```diff
@@ -0,0 +1,60 @@
+# Outline - "How to Find Coding Resources if You're New to Tech"
+
+### Step One
+
+**Title:**
+
+How to Find Coding Resources if You're New to Tech
+
+**Draft:**
+Why not dip your toes into the world of coding if you have some extra time on your hands? Whether you're bored during summer or want to learn new skills, learning a bit of coding is a great idea. Knowing how to code is invaluable knowledge, not matter what level of mastery you're at. And as a bonus, it gives you the ability to build and create awesome projects with more flair and creativity. 
+  
+**What is coding?**
+
+This might seem like a basic question, but if you're brand new to coding, its good to go over the basics! In sum, coding is using programming languages to tell a computer how to behave. As a coder, you're using the computer's language to make it do something. 
+
+This could be as simple as making your font bigger and bolder, or it could be more complicated, like coding a video game. 
+
+**What are some coding languages?**
```

Comment:
Can you find a visual to supplement this section and cite it at the bottom

Reply:
@shreyagupta98, visuals of different coding languages?